utils/docker/vcluster/README.md

```diff
@@ -0,0 +1,199 @@
+- `DAOS_ADMIN_USER`: Name or uid of the daos administrattor user (default "root")
+- `DAOS_ADMIN_GROUP`: Name or gid of the daos administrattor group (default "root")
+
+From a local tree, a more straightforward way to build these images could be done with
+`docker-compose` and the following commands:
+
+```bash
+$ cd utils/docker/vcluster
+$ docker-compose build
+```
+
+The same arguments are accepted but they have to be defined in the Docker Compose environment file
+`utils/docker/vcluster/.env`.
+
+!!! warning
+    For working properly, the DAOS authentication have to be enabled in all the images (i.e. nodes
+    images and base image).
+
+## Running DAOS Nodes Container
+
+### Starting Containers with docker command
+
+Once the images are created, the containers could be directly started with docker with the following
+commands:
+
+```bash
+$ export DAOS_IFACE_IP=x.x.x.x
+$ docker run --detach --privileged --name=daos-server --hostname=daos-server \
+	--add-host "daos-server:$DAOS_IFACE_IP" --add-host "daos-admin:$DAOS_IFACE_IP" \
+	--add-host "daos-client:$DAOS_IFACE_IP" --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
+	--volume=/dev/hugepages:/dev/hugepages  --tmpfs=/run --network=host \
+	daos-server:centos8
+$ docker run --detach --privileged --name=daos-agent --hostname=daos-agent \
+	--add-host "daos-server:$DAOS_IFACE_IP" --add-host "daos-admin:$DAOS_IFACE_IP" \
+	--add-host "daos-client:$DAOS_IFACE_IP" --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
+	--tmpfs=/run --network=host daos-agent:centos8
+$ docker run --detach --privileged --name=daos-client --hostname=daos-client \
+	--add-host "daos-server:$DAOS_IFACE_IP" --add-host "daos-admin:$DAOS_IFACE_IP" \
+	--add-host "daos-client:$DAOS_IFACE_IP" --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
+	--tmpfs=/run --network=host daos-client:centos8
+```
+
+The value of the `DAOS_IFACE_IP` shall be replaced with the one of the network interface which was
+provided when the images have been built.
+
+Once started, the DAOS server waits for the administrator to format the system.
+This can be done using the following command:
+
+```bash
+$ docker exec daos-admin dmg -i storage format
+```
+
+Upon successful completion of the format, the storage engine is started, and pools
+can be created using the daos admin tool.  For more advanced configurations and usage refer to the
+section [DAOS Tour](https://docs.daos.io/QSG/tour/).
+
+
+### Starting Containers with docker-compose command
+
+From a local tree, a more straightforward way to start the containers could be done with
+`docker-compose` and the following commands:
+
+```bash
+$ cd utils/docker/vcluster
+$ docker-compose up --detach
+```
+
+!!! note
+    Before starting the containers with `docker-compose`, the IP address of the network interface,
+    which was provided when the images have been built, shall be defined in the Docker
+    Compose environment file `utils/docker/vcluster/.env`.
```

Comment:
I seem to have missed this step, yet I was able to run fio anyway?

Reply:
Not needed if you use the daos-cm helper script.
Only needed, if you want to use directly docker-compose.